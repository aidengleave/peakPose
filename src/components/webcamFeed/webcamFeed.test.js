import React from "react";
import WebcamFeed from "./index.js";
import { shallow } from "enzyme";

describe("WebcamFeed", () => {
  it("should render correctly with no props", () => {
    const component = shallow(<WebcamFeed />);

    expect(component).toMatchSnapshot();
  });
});
