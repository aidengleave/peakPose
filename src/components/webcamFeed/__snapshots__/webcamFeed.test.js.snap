// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WebcamFeed should render correctly with no props 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": true,
        "rules": Array [
          "
  border: 5px solid palevioletred;
",
        ],
      },
      "displayName": "Styled(WebcamFeed)",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": [Function],
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
      Symbol(Symbol.hasInstance): [Function],
    }
  }
  forwardedRef={null}
/>
`;
